docs/content/Blankslate.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: Blankslate
+componentId: blankslate
+---
+
+import data from '../../src/Blankslate/Blankslate.docs.json'
+
+Use `Blankslate` when there is a lack of content within a page or section. Use as placeholder to tell users why something isn't there.
```

Comment:
If Blankslate is not being exported from any bundle, we should either not include it in the docs or add a blurb explaining why it's not in the main bundle.